### DISCUSSION TITLE
Sharp: Installation error: aborted

### DISCUSSION THREAD

**Discussion by moliter** [payloadcms/payload#4629] [Q&A]
Hi,
I have an error when installing payloadCMS. I tried to change the node version but nothing happening. I need some help. Thanks

**env**: wsl ubuntu22.04

**cmd**: npm install payload

**result**:
npm ERR! code 1
npm ERR! path /home/heriky/Code/next/digitalhippo/node_modules/payload/node_modules/sharp
npm ERR! command failed
npm ERR! command sh -c (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
npm ERR! sharp: Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.14.5/libvips-8.14.5-linux-x64.tar.br
npm ERR! sharp: Please see https://sharp.pixelplumbing.com/install for required dependencies
npm ERR! sharp: Installation error: aborted

**Comment by DanRibbens** [comment]
I always start with removing node_modules and package locks.

If that doesn't work you can try using the included docker configuration to rule out something specific to your environment.

Another option is to try a different package manager like pnpm or yarn 1.x

What version of node and npm are you on?

**Reply by moliter** [reply]
Hi, I used pnpm or yarn as my package manager, but the result is the same. I think this error is caused by my network because I'm in China. Thanks

node: v20.10.0
npm: 10.2.3

![image](https://github.com/payloadcms/payload/assets/37175477/d676dc24-bbe1-489f-9383-26e6764fb4d4)
